remotes/docker/config/hosts_test.go

```diff
@@ -104,6 +104,10 @@ ca = "/etc/path/default"
 
 [host."https://test-3.registry"]
   client = ["/etc/certs/client-1.pem", "/etc/certs/client-2.pem"]
+
+[host."https://noncompliantmirror.registry/v2/namespaceprefix"]
+  capabilities = ["pull"]
+  override_path = true
```

Comment:
maybe add a couple more tests for coverage:
```
[host."https://noncompliantmirrornilpath.registry"]
   override_path = true
[host."https://noncompliantmirrornilversion.registry/namespaceprefix"]
   override_path = true
```

or should these error out?

Reply:
Could add a case for the first one, the second two don't test anything different

Reply:
true was more anecdotal observation :-)